### PR TITLE
config: set default region size to 256M

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: etc/config-template.toml [hunk 1]
```diff
@@ -73,11 +73,11 @@ pd-store-heartbeat-tick-interval = "10s"
...
-region-split-size = "96MB"
+region-max-size = "384MB"
+region-split-size = "256MB"
 # When region size changes exceeds region-split-check-diff, we should check
 # whether the region should be split or not.
-region-split-check-diff = "12MB"
+region-split-check-diff = "32MB"
 
 # Interval to check region whether need to be split or not.
 split-region-check-tick-interval = "5s"
```
Comment (siddontang) [inline etc/config-template.toml]: do we need to increase the interval here?

Comment (BusyJay) [inline etc/config-template.toml]: No.

**Comment by siddontang** [comment]
PTAL @javaforfun @zhangjinpeng1987

**Comment by javaforfun** [comment]
LGTM

**Pull request merged**